2025-05-03 15:02:15,660 INFO: Application startup [in D:\do_an_tot_nghiep\thuc_hanh\backend\app.py:68]
2025-05-03 15:03:35,277 INFO: Application startup [in D:\do_an_tot_nghiep\thuc_hanh\backend\app.py:68]
2025-05-03 15:03:50,665 INFO: Application startup [in D:\do_an_tot_nghiep\thuc_hanh\backend\app.py:68]
2025-05-03 15:11:22,434 ERROR: Unhandled error: validate_file_extension() takes 1 positional argument but 2 were given [in D:\do_an_tot_nghiep\thuc_hanh\backend\middleware\error_handler.py:44]
2025-05-03 15:11:22,457 ERROR: Traceback (most recent call last):
  File "D:\do_an_tot_nghiep\thuc_hanh\backend\venv\lib\site-packages\flask\app.py", line 917, in full_dispatch_request
    rv = self.dispatch_request()
  File "D:\do_an_tot_nghiep\thuc_hanh\backend\venv\lib\site-packages\flask\app.py", line 902, in dispatch_request
    return self.ensure_sync(self.view_functions[rule.endpoint])(**view_args)  # type: ignore[no-any-return]
  File "D:\do_an_tot_nghiep\thuc_hanh\backend\utils\auth.py", line 47, in decorated
    return f(current_user, *args, **kwargs)
  File "D:\do_an_tot_nghiep\thuc_hanh\backend\middleware\rate_limit.py", line 63, in wrapped
    response = f(*args, **kwargs)
  File "D:\do_an_tot_nghiep\thuc_hanh\backend\routes\face_verification.py", line 27, in verify_face
    if not validate_file_extension(selfie_file.filename, current_app.config['ALLOWED_IMAGE_EXTENSIONS']):
TypeError: validate_file_extension() takes 1 positional argument but 2 were given
 [in D:\do_an_tot_nghiep\thuc_hanh\backend\middleware\error_handler.py:45]
2025-05-03 15:14:55,479 INFO: Application startup [in D:\do_an_tot_nghiep\thuc_hanh\backend\app.py:68]
2025-05-03 15:15:45,587 INFO: Application startup [in D:\do_an_tot_nghiep\thuc_hanh\backend\app.py:68]
2025-05-03 15:16:07,244 INFO: Application startup [in D:\do_an_tot_nghiep\thuc_hanh\backend\app.py:68]
2025-05-03 15:16:33,506 INFO: Application startup [in D:\do_an_tot_nghiep\thuc_hanh\backend\app.py:68]
2025-05-03 15:16:49,250 INFO: Application startup [in D:\do_an_tot_nghiep\thuc_hanh\backend\app.py:68]
2025-05-03 15:19:30,681 INFO: Application startup [in D:\do_an_tot_nghiep\thuc_hanh\backend\app.py:68]
2025-05-03 15:19:50,340 INFO: Application startup [in D:\do_an_tot_nghiep\thuc_hanh\backend\app.py:68]
2025-05-03 15:40:17,400 INFO: Application startup [in D:\do_an_tot_nghiep\thuc_hanh\backend\app.py:68]
2025-05-03 15:40:37,060 INFO: Application startup [in D:\do_an_tot_nghiep\thuc_hanh\backend\app.py:68]
2025-05-03 15:46:09,451 ERROR: Face verification error: Object of type bool_ is not JSON serializable [in D:\do_an_tot_nghiep\thuc_hanh\backend\routes\face_verification.py:86]
2025-05-03 15:53:16,818 INFO: Application startup [in D:\do_an_tot_nghiep\thuc_hanh\backend\app.py:68]
2025-05-03 15:53:34,070 INFO: Application startup [in D:\do_an_tot_nghiep\thuc_hanh\backend\app.py:68]
2025-05-03 15:57:32,130 INFO: Application startup [in D:\do_an_tot_nghiep\thuc_hanh\backend\app.py:68]
2025-05-03 15:57:53,934 INFO: Application startup [in D:\do_an_tot_nghiep\thuc_hanh\backend\app.py:68]
2025-05-03 15:59:09,325 INFO: Application startup [in D:\do_an_tot_nghiep\thuc_hanh\backend\app.py:68]
2025-05-03 15:59:38,914 INFO: Application startup [in D:\do_an_tot_nghiep\thuc_hanh\backend\app.py:68]
2025-05-03 16:00:06,838 INFO: Application startup [in D:\do_an_tot_nghiep\thuc_hanh\backend\app.py:68]
2025-05-03 16:00:38,869 INFO: Application startup [in D:\do_an_tot_nghiep\thuc_hanh\backend\app.py:68]
